src/types/datetime.rs

```diff
@@ -589,22 +593,26 @@ pub struct PyTzInfo(PyAny);
 pyobject_native_type!(
     PyTzInfo,
     crate::ffi::PyObject,
-    |py| ensure_datetime_api(py).TZInfoType,
+    |py| expect_datetime_api(py).TZInfoType,
     #module=Some("datetime"),
     #checkfunction=PyTZInfo_Check
 );
 
 /// Equivalent to `datetime.timezone.utc`
 pub fn timezone_utc(py: Python<'_>) -> &PyTzInfo {
-    unsafe { &*(ensure_datetime_api(py).TimeZone_UTC as *const PyTzInfo) }
+    unsafe {
+        &*(ensure_datetime_api(py)
```

Comment:
Why not call `expect_datetime_api` directly?

Reply:
Good catch, sorry. I just missed this during find-and-replace refactoring because of the line break introduced by rustfmt 👍